src/runtime/threads/coroutines/detail/context_generic_context.cpp

```diff
@@ -38,14 +38,9 @@ namespace hpx { namespace threads { namespace coroutines
             if (ctx_)
             {
                 increment_stack_recycle_count();
-#if BOOST_VERSION < 106100
-                ctx_ =
-                    boost::context::make_fcontext(stack_pointer_, stack_size_, funp_);
-#else
                 ctx_ =
```

Comment:
Have we dropped support for Boost V1.60 and below now?

Reply:
my bad ... those lines look almost the same...